### PR TITLE
[noise] Noise adjustments

### PR DESCRIPTION
**Pull request by NicEastvillage**
I adjusted the noise and the gradient weights slightly. The result is, that residential areas and industrial areas are slightly stronger.

Example:
esbjerg-noise

This PR depends on #65

### CODE REVIEW

**Comment by falkecarlsen** [comment]
Seems to depend on #65

**Comment by NicEastvillage** [comment]
> This PR depends on #65

yep

File: utility.py [hunk 1]
```diff
@@ -183,6 +183,10 @@ def setup_logging(args: dict):
     logger.addHandler(log_file_handler)
 
 
+def smoothstep(t: float):
+    return t * t * (3 - 2 * t)
```
Comment (falkecarlsen) [inline utility.py:187]: Are these values; `3` and `2` specific for our noise-values? If so, maybe a comment on it is appropriate.

File: perlin.py [hunk 2]
```diff
@@ -60,10 +60,10 @@ def sample_edge_noise(edge: sumolib.net.edge.Edge, base: int, centre,
     :return: the value between [0:1]
     """
     x, y = get_edge_pair_centroid(edge.getShape())
-    noise_value = get_perlin_noise(x, y, base=base, scale=scale, octaves=octaves)
-    gradient = (1 - (distance((x, y), centre) / radius)) * centre_weight
+    noise_value = get_perlin_noise(x, y, base=base, scale=scale, octaves=octaves) ** 1.3
```
Comment (falkecarlsen) [inline perlin.py]: I'm guessing `1.3` is just a voodoo param?

**Review by falkecarlsen — CHANGES_REQUESTED** [review]

**Comment by falkecarlsen** [comment]
Base branch is switched to make diff easier to read when depending on another PR. Remember to switch it back before merging.

File: utility.py (hunk 1 above)
Comment (NicEastvillage) [reply utility.py:187]: Nope. They are specific to the smoothstep function, and are the exact numbers needed to make the graph go through origin (0, 0) and the point (1, 1).

File: perlin.py (hunk 2 above)
Comment (NicEastvillage) [reply perlin.py]: Yes, but I actually wanted to remove that again since smoothstep did a better job. Removed in 06e31ce

**Review by falkecarlsen — APPROVED** [review]

**Pull request merged**